src/main/java/org/prebid/server/bidder/conversant/ConversantAdapter.java

```diff
@@ -78,19 +79,19 @@ private BidRequest createBidRequest(AdapterRequest adapterRequest, PreBidRequest
         final List<Imp> imps = makeImps(adUnitBidsWithParams, preBidRequestContext);
         validateImps(imps);
 
-        final Site site = makeSite(preBidRequestContext, adUnitBidsWithParams);
-        if (site == null) {
-            throw new PreBidException("Conversant doesn't support App requests");
+        final PreBidRequest preBidRequest = preBidRequestContext.getPreBidRequest();
+        final App app = preBidRequest.getApp();
+        if (app != null && StringUtils.isBlank(app.getId())) {
```

Comment:
1. Pls move this check in `parseAndValidateParams()` method.
2. Check the "Missing site id" exception - we need to verify site id only if site present.